tests/cases_BKP.tsv

```diff
@@ -0,0 +1,3 @@
+en	the walking dead	walking dead
+ga	Na daoine maithe	daoine maithe
+fr	plus ça change plus c'est la même chose	plus ça change plus c'est même chose
```

Comment:
Please add additional updated test cases.

Reply:
Now uses the big cases.tsv file.